lib/irb/inspector.rb

```diff
@@ -93,9 +94,21 @@ def init
     # Proc to call when the input is evaluated and output in irb.
     def inspect_value(v)
       @inspect.call(v)
-    rescue
-      puts "(Object doesn't support #inspect)"
-      ''
+    rescue => e
+      puts "An error occurred when inspecting the object: #{e.inspect}"
+
+      if KERNEL_INSPECT.respond_to?(:bind_call)
+        begin
+          puts "Result of Kernel#inspect: #{KERNEL_INSPECT.bind_call(v)}"
```

Comment:
Is there a reason to use `KERNEL_INSPECT.respon_to?(:bind_call)` and `KERNEL_INSPECT.bind_call(v)` instead of `KERNEL_INSPECT.bind(v).call`?

`bind(v).call` will work in ruby 2.6 and I think `respond_to?` check is not needed

Reply:
👍 updated